photon-lib/src/main/scala/com/linkedin/photon/ml/Types.scala

```diff
@@ -24,6 +24,16 @@ object Types {
   type SSV = org.apache.spark.mllib.linalg.SparseVector
   type SparkVector = org.apache.spark.mllib.linalg.Vector
 
+  // A "sample" is a training, validation or scoring vector of feature values
+  type UniqueSampleId = Long
+
+  // A "coordinate" is made up of several random effects
```

Comment:
It this ok:
`A CoordinateId is a fixed effect type, random effect type, or matrix factorization model type (e.g. "per-user", "per-song" ...)`